### PR TITLE
Move edit link markdown right under the content

### LINKED ISSUE #19070: GitHub edit link on markdown template creates excessive whitespace when prev link missing.
## Description
The GitHub edit link on the markdown template creates excessive whitespace when the prev link not present.

Related: https://github.com/gatsbyjs/gatsby/issues/19034

### Steps to reproduce
Visit https://www.gatsbyjs.org/tutorial/ and scroll to the bottom.

### Expected result
The link should appear more visually related to the content above, it seems out of place with so much extra spacing above.

### Actual result
image

### Environment
- Chrome 78
- No Gatsby environment involved.

### PR DESCRIPTION
**Pull request by rizqinizamil**
## Description

Move edit link markdown (_Edit this page on GitHub_) right under the content to make it more related with content above.

**Before**
before

**After**
after

## Related Issues

Fixes #19070

### CODE REVIEW

File: www/src/components/markdown-page-footer.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default class MarkdownPageFooter extends React.Component {
         <hr sx={{ display: `none` }} />
         {this.props.page && (
           <a
-            sx={{ variant: `links.muted`, mt: 9 }}
+            sx={{ variant: `links.muted`, mt: 6 }}
```
Comment (fk) [inline www/src/components/markdown-page-footer.js]: I'd stick to `9` here:

image

Comment (rizqinizamil) [reply www/src/components/markdown-page-footer.js]: Yes I agree. I'll change it back.

**Comment by sever1an** [comment]
The email form is included in mdx here, so the edit link would allow it to be edited. It might not be the best... To put It above, the email form would need to be rendered by the template or the edit link by mdx.

**Comment by sever1an** [comment]
Would it possibly be even better to put the email form under the navigation links?

File: www/src/components/markdown-page-footer.js (hunk 1 above)
Comment (fk) [reply www/src/components/markdown-page-footer.js]: 👍 🙏

**Comment by tesseralis** [comment]
Having the email form be right on top of the "Edit page" link does look a little weird, but we had that problem even beforehand. The main reason those forms are in MDX was so the pages that contain them could be in Markdown instead of JSX. Putting the email form as part of the template (and having `showEmailForm` in the frontmatter of pages that need it) should work too. But I don't think it's blocking for this PR, so maybe we can create another issue for that?

**Comment by fk** [comment]
👍

**Pull request merged**